tests/lib/wifi.py

```diff
@@ -219,3 +219,19 @@ def prepare(self):
         pass
     def cleanup(self):
         pass
+
+class wifi_client_stub():
+    def disable_and_enable_wifi(self, *args, **kwargs):
```

Comment:
Can we make this enable_wifi() and disable_wifi() - it should be more  generically useful.